misc_settings/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Misc Settings",
```

Comment:
Maybe also change this to 'Miscellaneous'. The technical module name `misc_settings` is good, though.